tasklist/common/src/main/java/io/camunda/tasklist/entities/ProcessEntity.java

```diff
@@ -107,8 +108,17 @@ public ProcessEntity setStartedByForm(boolean startedByForm) {
     return this;
   }
 
+  public String getBpmnXml() {
+    return bpmnXml;
+  }
+
+  public ProcessEntity setBpmnXml(final String bpmnXml) {
+    this.bpmnXml = bpmnXml;
```

Comment:
[q] What happen in case of malformed XML over here?

Reply:
This data is coming from Zeebe, so I assume it's not possible to deploy a malformed xml process.

In case it comes, currently some information would not be stored properly.